packages/fuels-core/src/code_gen/function_selector.rs

```diff
@@ -2,8 +2,8 @@ use crate::utils::first_four_bytes_of_sha256_hash;
 use crate::ByteArray;
 use fuels_types::param_types::ParamType;
 
-/// Given a `ABIFunction` will return a ByteArray representing the function
-/// selector as specified in the Fuel specs.
+/// Given a function name and its inputs  will return a ByteArray representing
+/// the function selector as specified in the Fuel specs.
 
 pub fn resolve_fn_selector(name: &str, inputs: &[ParamType]) -> ByteArray {
```

Comment:
```suggestion
/// Given a function name and its inputs  will return a ByteArray representing
/// the function selector as specified in the Fuel specs.
pub fn resolve_fn_selector(name: &str, inputs: &[ParamType]) -> ByteArray {
```

Shouldn't this comment be bound to `resolve_fn_selector`? Seems like there was a line between them.